src/styles/blocks/header.scss

```
@@ -39,4 +39,9 @@
     font-size: 12px;
     font-weight: 500;
   }
+
+  &__link:hover {
+    transition: color $delayValue;
+    color: #00acdc;
+  }
```

Put transition to origin element not hover to make animation smooth in both directions:
```suggestion
  &__link {
    transition: color $delayValue;  
    }
```